neptune/internal/backends/alpha_integration_backend.py

```diff
@@ -94,7 +97,7 @@ def _get_init_experiment_operations(self, entrypoint, params, tags) -> List[alph
 
         # Assign source entrypoint
         init_operations.append(alpha_operation.AssignString(
-            path=['source_code', 'entrypoint'],
+            path=alpha_path_utils.parse_path(f'{SOURCE_CODE_ATTRIBUTE_SPACE}entrypoint'),
```

Comment:
It would be good to have constant for `entrypoint` as well since it's already used in more then one place in code.

Reply:
> since it's already used in more then one place in code

Where?
If I've implemented 'entrypoint' constant in `neptune/internal/utils/alpha_integration.py`, the only place where I'd use it is here.

But I have nothing against  adding such constant, but then we should do the same for name.

I suggest
```python
SYSTEM_NAME_ATTRIBUTE_PATH = f'{SYSTEM_ATTRIBUTE_SPACE}name'
SOURCE_CODE_ENTRYPOINT_ATTRIBUTE_PATH = f'{SOURCE_CODE_ATTRIBUTE_SPACE}entrypoint'
```

Sounds good?